specification/applicationinsights/resource-manager/readme.md

```diff
@@ -222,6 +222,31 @@ directive:
   
 ```
 
+### Tag: package-2022-01-11
```

Comment:
>package-2022-01-11 [](http://example.com/codeflow?start=9&length=18)

we call every Tag "package-" but maybe we should have special tag if we plan to publish the sdk for it? either that or have appropriate comments so the next person doesn't overwrite or regress the versions. maybe all add links to "best practices" and other sdk architecture

• [https://github.com/Azure/azure-sdk-for-rust/issues/563](https://github.com/Azure/azure-sdk-for-rust/issues/563)
• [https://github.com/Azure/azure-sdk-for-rust/pull/567](https://github.com/Azure/azure-sdk-for-rust/pull/567)
Please also read these docs:
• [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/481/API-Spec-SDK-Package-Versioning-Considerations](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/481/API-Spec-SDK-Package-Versioning-Considerations)
• [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version)

Reply:
@vutran01 , is there another reason to declare a tag? I thought they were only for generating SDKs... I'll add the links, that's a good idea I think.

Reply:
yes, for building the sdk but not every tag is used to publish the sdk

Reply:
Added the links. Feel free to propose a convention, I don't think I have enough understanding of the overall process to have an opinion here. I don't even know why we would want to build an SDK and then not publish it.